src/content/blog/sweet-sprint-01.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Sweet Sprint Update 01"
+description: "Time for a do-over. I rewrite argument and file handling for typing exercise selection."
+startDate: "2024-10-02PDT"
+pubDate: "2024-10-02PDT"
+published: true
+heroImage: "/blog/sweet-sprint-01/cover.png"
+project: "sweet"
+tags:
+  - "go"
+---
+
+In case you didn't know, [sweet](https://github.com/NicksPatties/sweet) is the **S**oft**w**are **E**ngineer's **E**xercise for **T**yping I wrote last year. This is a command line application that's designed to help programmers practice typing. I've been thinking about some new features, and wanted to program in Go some more, so I started working on it again! I'm looking forward to getting this in the hands of programmers and getting some feedback.
+
+In the past two weeks, I made some changes to the project to improve my developer experience, but most importantly, I made improvements to selecting different exercises. Allow me to share what I've done!
+
+# An introduction
+
+Since this may be the first time you've heard of `sweet`, I'll share some quick details about how it works.
+
+Running the command opens a random exercise to practice. Type in the exercise, and once done, your stats appear on the screen, and the program ends.
+
+![Default behavior of sweet](@assets/blog/sweet-sprint-01/original-behavior.gif)
+
+## Why `sweet`?
```

Comment:
I don't think I need this section, but I can transition to the fixes section easier